### DISCUSSION TITLE
[Projects Beta] Allow columns to represent Milestones in Project Board view

### DISCUSSION THREAD

**Discussion by tboddyspargo** [community/community#11625] [Projects and Issues]
# Summary
I'd love to see the ability for the columns on a project board represent Milestones. Drag+drop a card between columns would remove the Milestone that the source column represented and add the Milestone that the target column represented. A default column could represent "no additional Milestone". Milestones other than the one represented by the source/target columns during a drag operation would not be modified.

**Comment by znorman-harris** [comment]
I also think this would be great. There's so many other ways to slice and dice the data in the board view, but not this. I'm wondering if there might be a better solution instead of creating a second field in the table view with a duplicate of the release milestone (that's what I did, not sure if I will stick with it or not yet).

**Comment by jbw976** [comment]
I would also love this feature - The only solution I can think of right now is creating a Milestone/Release custom field that is type single select, and then use that as the "column field", but that is essentially duplicating the built in Milestone value.  It would be better to be able to use Milestone as the column field on a project board.

**Comment by slam-campup** [comment]
Just ran into this issue this week — there's no way to generates a milestone-based issue board that allows the users to drag and drop issues into milestones.

**Comment by Wazbat** [comment]
I'm surprised this is still not possible. We're having to duplicate everything using a project board field to represent epics as well as milestones. It's very much a pain

**Comment by maximecolin** [comment]
I would love having this feature too. I use milestone to materialize iterations and it will be very convenient to create a board view with milestone as column and being able to move issues across milestones.

**Comment by zlove** [comment]
+1 for this. 

We use Milestones as a proxy for "Epics" and would like to be able to create Project Views with Milestones as cards in order to do higher-level (like quarterly / annual) planning.

**Comment by JoshHrach** [comment]
This is exactly what I'm looking for. I want to use Milestones as upcoming versions of my app. However, since I can't use Milestones as columns in a Board view, I end up creating a "App Version" enum that I use so I can move planned features between upcoming releases. Unfortunately, these single values don't have any due dates or progress charts, so I have no way of seeing my progress towards a release or an easy way to see what I want to release these updates.
